release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -529,6 +529,80 @@ For more information, see xref:../networking/advanced_networking/route_advertise
+
+* Azure Premium solid-state drives (SSD)
+
+* Standard SSDs
+
+* Standard hard disk drives (HDD)
+
+This feature is generally available in {product-title} 4.20.
+
+//For more information about performance plus, see ???
+
+[id="ocp-release-notes-storage-change-block-tracking_{context}"]
+==== Change block tracking (Developer Preview)
+Changed block tracking enables efficient and incremental backups and disaster recovery for persistent volumes (PVs) managed by Container Storage Interface (CSI) drivers. 
+
+Changed block tracking allows consumers to requests a list of blocks that have changed between two snapshots, which is useful for backup solutions vendors. By only backing up changed blocks, rather than entire volumes, back up processes are more efficient.
+
+:FeatureName: Change block tracking
+include::snippets/developer-preview.adoc[]
+
+[id="ocp-release-notes-storage-efs-zonal-vol-support_{context}"]
+==== AWS EFS One Zone volume support is generally available
+{product-title} 4.20 introduces AWS Elastic File Storage (EFS) One Zone volume support as generally available. With this feature, if file system Domain Name System (DNS) resolution fails, the EFS CSI driver can fall back to mount targets. A mount target serves as a network endpoint that allows AWS EC2 instances or other AWS compute instances within a Virtual Private Cloud (VPC) to connect to, and mount, an EFS file system.
+
+For more information about One Zone, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#efs-one-zone_persistent-storage-csi-aws-efs[Support for One Zone].
+
+[id="ocp-release-notes-storage-storage-security-policy_{context}"]
+==== Configuring fsGroupChangePolicy and selinuxChangePolicy ast namespace and pod level is generally available
+Certain characteristics of a volume can cause pod startup delays, which might cause pod timeouts.
+
+For volumes with a lot of files, pod startup timeouts can occur because, by default, {product-title} recursively changes ownership and permissions for the contents of each volume to match the `fsGroup` specified in a pod’s `securityContext` when that volume is mounted. This can be time consuming, slowing pod startup. You can use the `fsGroupChangePolicy` parameter inside a `securityContext` to control the way that {product-title} checks and manages ownership and permissions for a volume. 
+
+Changing this parameter at the pod level was introduced in {product-title} 4.10. In 4.20, you can set this parameter at the namespace level as well as pod level as a generally available feature.
+
+SELinux (Security-Enhanced Linux) is a security mechanism that assigns security labels (contexts) to all objects (files, processes, network ports, etc.) on a system. These labels determine what a process can access. When a pod starts, the container runtime recursively relabels all files on a volume to match a pod’s SELinux context. For large volumes, this can significantly increase pod startup times. 
```

Comment:
Same cause as FSGroup, this is not necessarily large volumes (in size) but volumes with a large number of files. We can keep the same wording "volumes with a lot of files"

Reply:
Done.